File: OtherComponents/MDWSvistalayer/MDWS Source/mdo/mdo/src/mdo/domain/sm/trash/PatientFacility.java
package gov.va.med.mhv.sm.model;

import java.io.Serializable;

public class PatientFacility extends BaseModel implements Serializable{

	/**
	 * 
	 */
	private static final long serialVersionUID = 66888812314511034L;
	private User user;
	private String stationNo;
	private String dfn;
	
	
	public User getUser() {
		return user;
	}
	public void setUser(User user) {
		this.user = user;
	}
	public String getStationNo() {
		return stationNo;
	}
	public void setStationNo(String stationNo) {
		this.stationNo = stationNo;
	}
	public String getDfn() {
		return dfn;
	}
	public void setDfn(String dfn) {
		this.dfn = dfn;
	}
	
	
}

File: OtherComponents/MDWSvistalayer/MDWS Source/mdo/mdo/src/mdo/domain/sm/PatientFacility.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace gov.va.medora.mdo.domain.sm
{
    [Serializable]
    public class PatientFacility : BaseModel
    {
        private User _user;

        public User User
        {
            get { return _user; }
            set { _user = value; }
        }
        private string _stationNo;

        public string StationNo
        {
            get { return _stationNo; }
            set { _stationNo = value; }
        }
        private string _dfn;

        public string Dfn
        {
            get { return _dfn; }
            set { _dfn = value; }
        }
    }
}
